project/Settings.scala

```diff
@@ -25,7 +25,7 @@ object Settings {
     val scalaXml          = "1.2.0"
     val catsTime          = "0.5.1"
 
-    val http4sVersion              = "0.23.24"
+    val http4sVersion              = "0.23.24-25-e71f1b1-SNAPSHOT"
```

Comment:
Why do we need a snapshot?

Reply:
The `Reconnect` middleware for WS is not released yet.